arenadata/scripts/run_resgroup_test.bash

```diff
@@ -7,6 +7,32 @@ function cleanup {
   docker-compose -p $project -f arenadata/docker-compose.yaml --env-file arenadata/.env down
 }
 
+function save_logs {
+  errcode=$?
+  docker-compose -p $project -f arenadata/docker-compose.yaml exec -T cdw bash -ex <<EOF
+      cd /home/gpadmin
+      tar -czf /logs/gpAdminLogs.tar.gz gpAdminLogs/
+      tar -czf /logs/gpAux.tar.gz gpdb_src/gpAux/gpdemo/datadirs/gpAdminLogs/
+      tar -czf /logs/pg_log.tar.gz gpdb_src/gpAux/gpdemo/datadirs/qddir/demoDataDir-1/pg_log/ gpdb_src/gpAux/gpdemo/datadirs/standby/pg_log
+      #regression.diffs may not exist if tests were successful
+      tar --ignore-failed-read -czf /logs/results.tar.gz gpdb_src/src/test/isolation2/results/resgroup/ gpdb_src/src/test/isolation2/regression.diffs
+EOF
+
+  docker-compose -p $project -f arenadata/docker-compose.yaml exec -T sdw1 bash -ex <<EOF
+      cd /home/gpadmin
+      tar -czf /logs/gpAdminLogs.tar.gz gpAdminLogs/
+      tar -czf /logs/gpAux.tar.gz gpdb_src/gpAux/gpdemo/datadirs/gpAdminLogs/
+      tar -czf /logs/pg_log.tar.gz \
+        gpdb_src/gpAux/gpdemo/datadirs/dbfast1/demoDataDir0/pg_log \
+        gpdb_src/gpAux/gpdemo/datadirs/dbfast2/demoDataDir1/pg_log \
+        gpdb_src/gpAux/gpdemo/datadirs/dbfast3/demoDataDir2/pg_log \
+        gpdb_src/gpAux/gpdemo/datadirs/dbfast_mirror1/demoDataDir0/pg_log \
+        gpdb_src/gpAux/gpdemo/datadirs/dbfast_mirror2/demoDataDir1/pg_log \
+        gpdb_src/gpAux/gpdemo/datadirs/dbfast_mirror3/demoDataDir2/pg_log
+EOF
```

Comment:
It is okay for the default demo cluster configuration. Do we consider different configurations with less of more segments, without mirrors, etc?

Reply:
All tests rely on the configuration of the master + 3 segments + 3 mirrors. I am not sure that the tests will pass in a different configuration. Tests will definitely not pass without mirrors (verified).